provisioning/miniprov/hare_mp/main.py

```diff
@@ -603,7 +631,11 @@ def kv_cleanup():
         KeyDelete(name='m0conf/', recurse=True),
         KeyDelete(name='processes/', recurse=True),
         KeyDelete(name='stats/', recurse=True),
-        KeyDelete(name='mkfs/', recurse=True)
+        KeyDelete(name='mkfs/', recurse=True),
+        KeyDelete(name='bytecount/', recurse=True),
+        KeyDelete(name='config_path', recurse=True),
```

Comment:
Why do we have recurse=True here and recurse=False for 'failvec'?

Reply:
We have only one key named by `failvec`. So I tried to delete only one time not recursively.
```
[root@ssc-vm-g2-rhev4-1947 hare]# consul kv get -recurse
bytecount/critical:0
bytecount/damaged:0
bytecount/degraded:0
bytecount/healthy:0
config_path:yaml:///root/kuber/single_node_confnew.yaml
failvec:
m0_client_types:[]
```

Reply:
`We have only one key named by failvec`
What about 'config_path'?

Reply:
Okay, Let me change for `failvec` also.

Reply:
Addressed.